### PR TITLE
Migrate remaining linters to pre-commit

### PR DESCRIPTION
**Pull request by amacneil**
**Public API Changes**
None

**Description**
Follow up from https://github.com/RobotWebTools/rosbridge_suite/pull/648:

- Remove separate `pre-commit.yaml` Github workflow in favor of existing `ci.yml` workflow
- Removed unused json and toml linters from `.pre-commit-config.yaml`
- `isort` is updated to v5.9.3
- `bandit`, `codespell`, and `flake8` are migrated to `.pre-commit-config.yaml` so all linters are in one place
- Fixed wildcard and invalid imports in `rosapi_node` which were missed by https://github.com/RobotWebTools/rosbridge_suite/pull/609 (due to not having a .py extension)

Related PR where this was originally suggested (fyi @cclauss): https://github.com/RobotWebTools/rosbridge_suite/pull/630

cc @kenji-miyake

### CODE REVIEW

File: rosapi/scripts/rosapi_node [hunk 1]
```diff
@@ -277,12 +303,12 @@ class Rosapi(Node):
 def get_service_host(request):
```
Comment (amacneil) [inline rosapi/scripts/rosapi_node:303]: note the comment above this line indicates that this service isn't used, so I'm not sure what to make of it

**Review by cclauss — APPROVED** [review]
LGTM

File: rosapi/scripts/rosapi_node (hunk 1 above)
Comment (jtbandes) [reply rosapi/scripts/rosapi_node:303]: Probably best to comment out or delete the code if it's not working 🤷

**Review by jtbandes — APPROVED** [review]

File: rosapi/scripts/rosapi_node (hunk 1 above)
Comment (amacneil) [reply rosapi/scripts/rosapi_node:303]: Agreed, although I don't actually know what this code does so was erring on the safe side.

I will make a separate PR to remove it.

**Pull request merged**